python/tvm/relay/op/strategy/x86.py

```diff
@@ -125,6 +126,32 @@ def conv2d_strategy_cpu(attrs, inputs, out_type, target):
                 wrap_topi_schedule(topi.x86.schedule_conv2d_nhwc),
                 name="conv2d_nhwc.x86",
             )
+
+            judge_winograd_auto_scheduler = False
+            if len(kernel.shape) == 4:
+                kernel_h, kernel_w, _, co = get_const_tuple(kernel.shape)
+                judge_winograd_auto_scheduler = (
+                    "float" in data.dtype
+                    and "float" in kernel.dtype
+                    and kernel_h == 3
+                    and kernel_w == 3
+                    and stride_h == 1
+                    and stride_w == 1
+                    and dilation_h == 1
+                    and dilation_w == 1
+                    and 64 < co < 512
```

Comment:
I think we could add one comment that we find we only work more effective for `64 < co < 512` of resnet18 network on our skylake machine. But this could prompt others maybe you could adjust this condition on your target machine. Otherwise this will bring in some confuse why there is such one condition here.